packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -214,6 +215,13 @@ export class EntityService {
     args: CreateOneEntityArgs,
     user: User
   ): Promise<Entity> {
+
+    if (args.data?.name?.toLowerCase() === args.data?.pluralDisplayName?.toLowerCase()) { 
+      throw new AmplicationError(
+        `The entity name must be in the singular form.`
```

Comment:
Use this message instead "the entity name and plural display name cannot be the same"